src/components/tabController/apis/tabController.api.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "TabController",
+  "category": "layoutsAndTemplates",
+  "description": "A performant solution for a tab controller with lazy load mechanism",
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/TabControllerScreen/index.tsx",
+  "note": "This component is based on react-native-gesture-handler.\nOn Android, when using react-native-navigation, make sure to wrap your screen with gestureHandlerRootHOC",
```

Comment:
Shouldn't it be "notes" (plural)?

Reply:
You said yesterday we're using `note` cause we're noting the user 
It's just a big note (:

Reply:
I thought on component-level it's "notes" and on prop-level it's "note". But ok. I'll change it on my side as well